### DISCUSSION TITLE
Requests tab filter (upcomming)

### DISCUSSION THREAD

**Discussion by drohack** [sct/overseerr#3954] [Feature Requests]
It would be really nice to have a filter on the Requests tab that hides all of the requests of shows/movies that have not aired yet.

I frequently use the Requests tab to manually search for items that can't be auto acquired, and having to scroll through all of the future movies/shows each time is a bit tiresome. Also it doesn't show their air date, just the year, so it's sometimes hard to tell if something has aired this year yet or not.

Not sure if this would be a different request, but the options I'd like to filter on are:
- Available (the digital release date has passed)
- Media type (Movie, TV Show, etc.)
- Request date (what time someone requested the item, i.e. older than 30 days)
- Request status (Requested, Partially Available)

**Comment by Lockszmith-GH** [comment]
Seems related to #3335

**Reply by drohack** [reply]
Yeah that seemed a little more wide spread where that tag shows up everywhere, not just the Requests tab. And that requests potentially asking for it to be more defined (show more than just Available, and Partially Available). But def slightly related.
